src/Common/src/Interop/Unix/System.Native/Interop.MkDir.cs

```diff
@@ -4,13 +4,11 @@
 using System;
 using System.Runtime.InteropServices;
 
-using mode_t = System.Int32;
-
 internal static partial class Interop
 {
-    internal static partial class libc
+    internal static partial class Sys
     {
         [DllImport(Libraries.Libc, SetLastError = true)]
```

Comment:
This needs to be SystemNative not libc and CI is failing. Waiting for full report to make sure there's nothing else before resetting CI with that fix.